plugins/modules/aci_l3out_extepg_to_contract.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, fn ln (@kudtarkar1) <@.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l3out_extepg_to_contract:
+short_descrption: Bind External End Point Groups to Contracts 
+descrption: 
+- Bind ExtEPGs to Contracts on ACI fabrics.
+Version_added: 
+Options:
+  tenant:
+    descrption:
+    - Name of existing tenant.
+    type: str
+    aliases: [tenant_name]
+  l3Out:
+    descrption:
+    - The name of the L3Outs
+    type: str
+    aliases: [l3out_name]
+  extepg:
+    descrption:
+    - The name of the external end point groups
+    type: str
+    aliases: [extepg_name]
+  contract:
+    descrption:
+    - The name of the contract
+    type: str
+  contract_type:
+    descrption:
+    - The type of contract(provider or consumer)
+  state:
+    descrption:
+    - Use pesent or absent for adding and removing
+    - Use query for listing an object or multiple ojbects
+    type: str
+    choices: [present, absent, query]
+extends_documentation_fragment: aci
+note:
+- The tenant, l3Out, extepg, contract should exist before using this module
```

Comment:
Does the contract need to exist? I don't think so. I would change this to:
- The external EPG (extepg) should exist before using this module.
As extepg requires l3out which in turn requires tenant, we don't need to say more.